presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -1533,16 +1539,6 @@ private static Object optimize(@Language("SQL") String expression)
             }
             return value;
         });
-        RowExpression rowExpression = TRANSLATOR.translate(parsedExpression, SYMBOL_TYPES);
```

Comment:
cc @highker on these removal

Reply:
This is not present in PrestoSql repo. Having this throws an exception during translation OperatorNotFoundException: '[]' cannot be applied to row(integer,varchar(1),boolean), integer.
Let me know how to go about it.

Reply:
We need to keep this. The problem is `SqlToRowExpressionTranslator` directly calls the subscript operator without desugaring that. For all SQL sugars that need to be desugared, we might either add the same logic to `SqlToRowExpressionTranslator` (like, `currentPath` or `currentUser`, which are very straightforward) or skip that for the test given it is mostly a desugar test. the row subscript desugaring has some logic in it. It might be good to avoid having it in the translator. @wenleix, what do you think?

Reply:
Per offline discussion,  we shouldn't remove this part of code. Let's comment out line 1432, so the tests won't run, and add a comment like:
```
// @Test
// TODO: enable the test once desugar supports RowExpression
```